src/server/routes/register.ts

```diff
@@ -68,102 +68,11 @@ router.post(
   '/register/email-sent/resend',
   handleRecaptcha,
   handleAsyncErrors(async (req: Request, res: ResponseWithRequestState) => {
-    const { useOkta } = res.locals.queryParams;
-    if (okta.enabled && useOkta) {
-      await OktaResendEmail(req, res);
+    const { useIdapi } = res.locals.queryParams;
+    if (useIdapi) {
```

Comment:
probably a bit early to review this pr, but it will be useful to keep the existing switch in `okta.enabled` so we can disable okta through a switch

so we should do something like
```suggestion
    if (!okta.enabled || useIdapi) {
```

Reply:
Good idea!  Done in 3b71c141d700091db6e04b7613c69a6e932e747b